README.md

```diff
@@ -52,3 +60,7 @@ VITE_FIREBASE_STORAGEBUCKET=
 VITE_FIREBASE_MESSAGINGSENDERID=
 VITE_FIREBASE_APPID=
 ```
+
+Also remember to enable Google Authentication as provider in your firebase project. You can do this by going to the Authentication tab in the firebase console.
+
```

Comment:
You've set this up more recently than I have. Did you need to do anything with Firestore or Storage to set those components up?

Reply:
I did not used storage at the moment, but I needed to enable google authentication manually otherwise the login modal fails and can not login

Reply:
Ah I see. Thanks!